src/resolvers/Query.js

```diff
@@ -0,0 +1,10 @@
+const Query = {
+  info: () => `This is the API of a Reporting System`,
+  divisions: (root, args, context, info) => {
```

Comment:
use destructuring for context here, and if you use param that named parent above, it should be parent here too.